content/en/docs/components/hyperparameter-tuning/resume-experiment.md

```diff
@@ -16,8 +17,9 @@ For details of how to configure and run your experiment, see the guide to
 
 ## Modify running experiment
 
-While experiment is running you are able to change trial count parameters.
-For example, if you want to decrease maximum number of hyperparameters sets that are trained parallel.
+While the experiment is running you are able to change trial count parameters.
+For example, if you want to decrease the maximum number of
+hyperparameter sets that are trained parallel.
```

Comment:
This may appear like an incomplete sentence because of the "if" statement. Let's try the following:

```suggestion
While the experiment is running you are able to change trial count parameters.
For example, you can decrease the maximum number of
hyperparameter sets that are trained in parallel.
```

Note: hyperparam sets can be trained "in parallel" not "parallel", I think.